its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M6/values.yaml

```diff
@@ -0,0 +1,2 @@
+escalation: true
+#           ^^^^   #TODO: SONARIAC-1132 Raise here the secondary location
```

Comment:
It should be here SONARIAC-1131 Raise a secondary location from where the sensitive value comes from

Reply:
Changed